src/main/thrift/parquet.thrift

```diff
@@ -863,23 +869,27 @@ struct ColumnIndex {
 }
 
 struct AesGcmV1 {
```

Comment:
AesGcmV1 and AesGcmCtrV1 seems identical. If they are same, can we just use one structure instead of duplicating?

Reply:
- we use them to specify two different algorithms. since we dont use enums, its a union with two structs
- the design is signed off. we shouldn't change it, lets only make sure the Thrift pr conforms to the merged design.

Reply:
We don't need to change the design. We can just define a sub-structure something like AadCollection which includes the 3 fields. And then both AesGcmV1 and AesGcmCtrV1 includes this sub-structure.